app/cafe.py

```diff
@@ -0,0 +1,21 @@
+from datetime import date
+
+from app.errors import (
+    NotWearingMaskError,
+    NotVaccinatedError,
+    OutdatedVaccineError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("The visitor is not vaccinated.")
+        if visitor["vaccine"]["expiration_date"] < date.today():
+            raise OutdatedVaccineError("The vaccine is expired")
+        if not visitor["wearing_a_mask"]:
```

Comment:
use get() method instead of parenthesis everywhere for getting value by key to avoid KeyError